templates/activity/parts/summon-profiles.hbs

```diff
@@ -0,0 +1,80 @@
+<section class="tab activity-{{ tab.id }} {{ tab.cssClass }}" data-tab="{{ tab.id }}" data-group="{{ tab.group }}">
+    <fieldset>
+        <legend>{{ localize "DND5E.SUMMON.FIELDS.profiles.label" }}</legend>
+        {{#with fields.summon.fields as |fields|}}
+        {{ formField fields.mode value=../source.summon.mode options=../profileModes }}
+        {{#unless activity.isSpell}}{{ formField fields.identifier value=../source.summon.identifier }}{{/unless}}
+        {{/with}}
+        <ul class="separated-list flexcol">
+            {{#each profiles}}
+            <li data-index="{{ index }}" data-profile-id="{{ data._id }}">
+                {{> ".profile" }}
+            </li>
+            {{else}}
+            <li class="empty">
+                {{#unless source.summon.mode}}{{ localize "DND5E.SUMMON.Profile.EmptyDrop"}}
+                {{~else}}{{ localize "DND5E.SUMMON.Profile.Empty" }}{{/unless}}
+            </li>
+            {{/each}}
+        </ul>
+        <button type="button" class="unbutton" data-action="addProfile">
+            {{ localize "DND5E.SUMMON.Profile.Action.Create" }}
+        </button>
+    </fieldset>
+</section>
+
+{{#*inline ".profile"}}
+    <div class="details flexrow">
+        {{ formInput fields.count name=(concat prefix "count") value=data.count placeholder="1" }}
+        <!-- aria-label="{{ localize 'DND5E.Summoning.Count.Label' }}" -->
```

Comment:
I left these in comments here because I couldn't find a way to attach them to fields using this helper.